docs/migration-2.md

```diff
@@ -0,0 +1,120 @@
+---
+comments: true
+---
+
+# Migrating from `g2p` 1.x
+
+The `g2p` 2.0 release introduces a number of improvements and changes
+which, unfortunately, are incompatible with mappings and Python code
+written for the previous version.  We have tried to describe them here
+with the changes you will need to make to your code and data.
+
+## Mapping configurations have changed (for the better)
+
+The configurations for mappings (which you'll find in
+`g2p/mappings/langs/*/config-g2p.yaml`) are now validated with a [YAML
+Schema](https://raw.githubusercontent.com/roedoejet/g2p/main/g2p/mappings/.schema/g2p-config-schema-2.0.json).
+If you use an editor like [Visual Studio
```

Comment:
I think the [yaml extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) needs to be installed to have it work on VS code